From d1061c48c8484f3775b6de1d2425b1406eda4d44 Mon Sep 17 00:00:00 2001
From: Yehuda Yitschak <yehuday@marvell.com>
Date: Mon, 22 Feb 2016 14:03:08 +0200
Subject: [PATCH 036/239] make: doimage: uncomment NAND paramaters to enable
 NAND boot

This has no effect on other boot modes since these flags are ignored
for non-NAND boot

Change-Id: Id39e1ed1faff28999ef5e784ea1437f66c1e106e
Signed-off-by: Yehuda Yitschak <yehuday@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/27739
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Haim Boot <hayim@marvell.com>
---
 tools/doimage/doimage.mk | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/tools/doimage/doimage.mk b/tools/doimage/doimage.mk
index bff4403..49aca9e 100644
--- a/tools/doimage/doimage.mk
+++ b/tools/doimage/doimage.mk
@@ -32,6 +32,6 @@ DOIMAGE_FLAGS		?= 	-l 0x4100000 -e 0x4100000
 #NAND params
 #Open and update the below when using NAND as a boot device.
 
-#CONFIG_MVEBU_NAND_BLOCK_SIZE	:=	128K
-#CONFIG_MVEBU_NAND_CELL_TYPE	:=	MLC
-#NAND_DOIMAGE_FLAGS     	:=      -t $(CONFIG_MVEBU_NAND_CELL_TYPE) -n $(CONFIG_MVEBU_NAND_BLOCK_SIZE)
+CONFIG_MVEBU_NAND_BLOCK_SIZE	:= 256
+CONFIG_MVEBU_NAND_CELL_TYPE	:= SLC
+NAND_DOIMAGE_FLAGS := -t $(CONFIG_MVEBU_NAND_CELL_TYPE) -n $(CONFIG_MVEBU_NAND_BLOCK_SIZE)
-- 
1.9.1

